openedx/adg/common/lib/mandrill_client/client.py

```diff
@@ -65,6 +71,8 @@ def send_mandrill_email(self, template, email, context):
             None
         """
         log.info(f'Sending email using template: {template}, account: {email} and context: {context} using mandrill')
-
+        user_lang = get_language()
+        if user_lang != settings.LANGUAGE_CODE:
+            template += '-' + user_lang
```

Comment:
How about moving this to a separate private method in the same class and giving that method a meaningful name? and calling here?